.github/workflows/python-package.yml

```
@@ -4,21 +4,35 @@ on:
   pull_request:
   release:
     types: [released]
+  workflow_dispatch:
+    inputs:
+      python_version:
+        default: 3.10
```

I'm not sure if it's a problem here, but yaml parsing (I've run into this elsewhere) might handle this first as a number and truncate it to 3.1 before the GitHub action converts it to a string. Best to wrap in quotes to be safe.
```suggestion
        default: `3.10`
```